### DISCUSSION TITLE
Onde é definido que o alertmanager deve ser um statefulset?

### DISCUSSION THREAD

**Discussion by matheus-yaml** [prometheus-operator/kube-prometheus#834] [Q&A] [answered]
Onde é definido que o alertmanager deve ser um statefulset?

Where is it defined that the alertmanager should be a statefulset?

**Comment by paulfantom** [comment, marked as answer]
Alertmanager statefulset is created by prometheus-operator from Alertmanager Custom Resource. The fact that it is a StatefulSet is defined in prometheus-operator code.

**Reply by matheus-yaml** [reply]
I am not found, can you help me? I am searching in > kube-prometheus/manifests/setup/prometheus-operator-0alertmanagerCustomResourceDefinition.yaml

**Reply by paulfantom** [reply]
The manifest is not in the repository as it is created in runtime by prometheus-operator. Operator takes values from Prometheus CR and creates statefulset. More on how prometheus-operator works can be found in https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md
